src/com/google/javascript/jscomp/PolymerClassDefinition.java

```diff
@@ -33,6 +33,8 @@
  * class.
  */
 final class PolymerClassDefinition {
+  /** Either 1 or 2 - the Polymer version of the definition */
+  final int version;
```

Comment:
yeah, there can be only one version of Polymer in a program, so that makes things very clear